### ISSUE TITLE
Broken build on arm64

### ISSUE THREAD

**Issue by smira** [containerd/continuity#168]
```
#162 36.67 /go/pkg/mod/github.com/containerd/continuity@v0.0.0-20201119173150-04c754faca46/fs/du_unix.go:97:25: invalid operation: stat.Blocks * stat.Blksize (mismatched types int64 and int32)
```

**Comment by smira** [comment]
Introduced in https://github.com/containerd/continuity/commit/bc5e3edd2b742c38c762d928f267ad82922a1b63

cc @dmcgowan

**Comment by smira** [comment]
Looks like there's a PR #165 for it

**Comment by estesp** [comment]
Closed by #165
